internal/controllers/gateway/gatewayclass_controller.go

```diff
@@ -115,7 +115,7 @@ func (r *GatewayClassReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	gwc := new(gatewayapi.GatewayClass)
 	if err := r.Client.Get(ctx, req.NamespacedName, gwc); err != nil {
 		if apierrors.IsNotFound(err) {
-			log.V(util.DebugLevel).Info("object enqueued no longer exists, skipping", "name", req.Name)
+			log.V(util.DebugLevel).Info("Object enqueued no longer exists, skipping", "name", req.Name)
```

Comment:
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/controllers/gateway/gatewayclass_controller.go#L94
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/controllers/gateway/gatewayclass_controller.go#L124
These are not changed.